Containerfile

```diff
@@ -23,33 +23,21 @@ RUN chown -R root:root /etc/squid/squid.conf /var/log/squid /var/spool/squid /ru
 FROM registry.access.redhat.com/ubi10/ubi-minimal@sha256:a13cec4e2e30fa2ca6468d474d02eb349200dc4a831c8c93f05a2154c559f09b AS go-builder
 
 # Install required packages for Go build
+# go-toolset already declared in rpms.in.yaml (prefetched by Cachi2)
 RUN if [ -f /cachi2/cachi2.env ]; then . /cachi2/cachi2.env; fi && \
     microdnf install -y \
     tar \
     gzip \
     gcc \
     curl \
     ca-certificates \
-    git && \
+    git \
+    go-toolset && \
```

Comment:
Did you also explore using an internal [golang base image](https://catalog.redhat.com/en/software/containers/ubi10/go-toolset/677d3db01d4db033edfda594#overview)? If that works, we don't need to install or track go-related stuff for this.